sway/commands/resize.c

```diff
@@ -280,10 +288,10 @@ static struct cmd_results *resize_adjust_tiled(uint32_t axis,
 		}
 	}
 
-	double old_width = current->width;
-	double old_height = current->height;
+	double old_width = current->width_fraction;
+	double old_height = current->height_fraction;
 	container_resize_tiled(current, axis, amount->amount);
-	if (current->width == old_width && current->height == old_height) {
+	if (current->width_fraction == old_width && current->height_fraction == old_height) {
```

Comment:
style: can you split this after the `&&` so that it's within 80 characters?

Reply:
Done and I think I followed the style now. Only one extra tab to indent a second line instead of spaces to align with the previous line. I also generally use 2 space tabs so my editor was giving me more room to fit in 80 columns than should be the reference with 4 spaces.

Reply:
It actually needs an additional indentation level here to differentiate it from the body

Reply:
Yep, misread that in `CONTRIUTING.md`. It's fixed now.